### ISSUE TITLE
Unable to debug in vscode

### ISSUE THREAD

**Issue by yunxdai** [nest/nest-simulator#1619]
**Describe the bug**
I installed NEST using conda and tried to develop my program using NEST in vscode. As I selected the right python interpreter, I can run my code in terminal. However, it reported the following error message when I wanted to "launch" and "debug" the same code.

```
python: /home/conda/feedstock_root/build_artifacts/nest-simulator_1569564426741/work/sli/scanner.cc:581: bool Scanner::operator()(Token&): Assertion `in->good()' failed.
[yunxdai-Ubuntu:03342] *** Process received signal ***
[yunxdai-Ubuntu:03342] Signal: Aborted (6)
[yunxdai-Ubuntu:03342] Signal code:  (-6)
[yunxdai-Ubuntu:03342] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x11390)[0x7f3561a17390]
[yunxdai-Ubuntu:03342] [ 1] /lib/x86_64-linux-gnu/libc.so.6(gsignal+0x38)[0x7f3561671428]
[yunxdai-Ubuntu:03342] [ 2] /lib/x86_64-linux-gnu/libc.so.6(abort+0x16a)[0x7f356167302a]
[yunxdai-Ubuntu:03342] [ 3] /lib/x86_64-linux-gnu/libc.so.6(+0x2dbd7)[0x7f3561669bd7]
[yunxdai-Ubuntu:03342] [ 4] /lib/x86_64-linux-gnu/libc.so.6(+0x2dc82)[0x7f3561669c82]
[yunxdai-Ubuntu:03342] [ 5] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libsli.so(_ZN7ScannerclER5Token+0x1489)[0x7f3533e5feb9]
[yunxdai-Ubuntu:03342] [ 6] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libsli.so(_ZN6ParserclER5Token+0x49)[0x7f3533e52229]
[yunxdai-Ubuntu:03342] [ 7] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libsli.so(_ZNK14IparseFunction7executeEP14SLIInterpreter+0x96)[0x7f3533e89666]
[yunxdai-Ubuntu:03342] [ 8] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libsli.so(+0x74193)[0x7f3533e48193]
[yunxdai-Ubuntu:03342] [ 9] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libsli.so(_ZN14SLIInterpreter8execute_Em+0x222)[0x7f3533e4ca32]
[yunxdai-Ubuntu:03342] [10] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libsli.so(_ZN14SLIInterpreter7startupEv+0x27)[0x7f3533e4ce57]
[yunxdai-Ubuntu:03342] [11] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/../../../libnest.so(_Z11neststartupPiPPPcR14SLIInterpreterNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0x1ea0)[0x7f35348ec030]
[yunxdai-Ubuntu:03342] [12] /home/yunxdai/anaconda3/envs/nest/lib/python3.6/site-packages/nest/pynestkernel.so(+0x419c5)[0x7f3534c8e9c5]
[yunxdai-Ubuntu:03342] [13] /home/yunxdai/anaconda3/envs/nest/bin/python(_PyCFunction_FastCallDict+0x154)[0x55c94d877b44]
[yunxdai-Ubuntu:03342] [14] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x1a1740)[0x55c94d905740]
[yunxdai-Ubuntu:03342] [15] /home/yunxdai/anaconda3/envs/nest/bin/python(_PyEval_EvalFrameDefault+0x30a)[0x55c94d92a7aa]
[yunxdai-Ubuntu:03342] [16] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x171a5b)[0x55c94d8d5a5b]
[yunxdai-Ubuntu:03342] [17] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x1a1635)[0x55c94d905635]
[yunxdai-Ubuntu:03342] [18] /home/yunxdai/anaconda3/envs/nest/bin/python(_PyEval_EvalFrameDefault+0x30a)[0x55c94d92a7aa]
[yunxdai-Ubuntu:03342] [19] /home/yunxdai/anaconda3/envs/nest/bin/python(PyEval_EvalCodeEx+0x329)[0x55c94d8da069]
[yunxdai-Ubuntu:03342] [20] /home/yunxdai/anaconda3/envs/nest/bin/python(PyEval_EvalCode+0x1c)[0x55c94d8dadfc]
[yunxdai-Ubuntu:03342] [21] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x1c40ed)[0x55c94d9280ed]
[yunxdai-Ubuntu:03342] [22] /home/yunxdai/anaconda3/envs/nest/bin/python(PyCFunction_Call+0x131)[0x55c94d87ab91]
[yunxdai-Ubuntu:03342] [23] /home/yunxdai/anaconda3/envs/nest/bin/python(_PyEval_EvalFrameDefault+0x539a)[0x55c94d92f83a]
[yunxdai-Ubuntu:03342] [24] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x170cf6)[0x55c94d8d4cf6]
[yunxdai-Ubuntu:03342] [25] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x171c91)[0x55c94d8d5c91]
[yunxdai-Ubuntu:03342] [26] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x1a1635)[0x55c94d905635]
[yunxdai-Ubuntu:03342] [27] /home/yunxdai/anaconda3/envs/nest/bin/python(_PyEval_EvalFrameDefault+0x30a)[0x55c94d92a7aa]
[yunxdai-Ubuntu:03342] [28] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x171a5b)[0x55c94d8d5a5b]
[yunxdai-Ubuntu:03342] [29] /home/yunxdai/anaconda3/envs/nest/bin/python(+0x1a1635)[0x55c94d905635]
[yunxdai-Ubuntu:03342] *** End of error message ***
```
I don't know why NEST simulator works when I directly started it in terminal but failed when I tried to "launch" it in vscode?

**Comment by gtrensch** [comment]
We've looked at VS Code and I'm afraid we can't do that much for you as this is not a NEST issue and is therefore out of our control.

**Comment by gtrensch** [comment]
We will improve the NEST documentation. This is addressed now in #1984 _"Add instructions for how to work with VS Code when developing NEST"_.

**Comment by gtrensch** [comment]
Solved by pull request #1984.
We have added instructions to the documentation for how to work with VS Code when developing NEST.

**Comment by gtrensch** [comment]
Issue closed.
